tools/autograd/gen_variable_type.py

```diff
@@ -159,6 +171,19 @@ def should_trace(declaration):
     base_name = name[:-1] if declaration['inplace'] else name[:-4] if name.endswith('_out') else name
     if base_name in DONT_RECORD_TRACE:
         return False
+    # We need to disable these because their inner implementations implement
```

Comment:
what's the plan moving forward with this?  How difficult is it to do something like 'we know this is a native function calling into a non-native function that does broadcasting...'.

Reply:
Also #8495